src/components/Join/Join.tsx

```diff
@@ -0,0 +1,179 @@
+import Modal from 'react-modal'
+import styles from './Join.module.scss'
+
+const tabs = [
+  {
+    id: 'show-options',
+    label: 'Show me some providers!',
+    content: (
+      <>
+        Here are some Pod providers that work with sleepy.bike:
+        <ul>
+          {oidcIssuers.map(({ issuer, registration }) => (
+            <li>
+              <ExternalButtonLink
+                secondary
+                href={registration}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {new URL(issuer).hostname}
+              </ExternalButtonLink>
+            </li>
+          ))}
+        </ul>
+      </>
+    ),
+  },
+  {
+    id: 'choose-for-me',
+    label: 'Choose for me!',
+    content: (
+      <>
+        <ExternalButtonLink
+          secondary
+          href="https://solidweb.me/idp/register/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Get a Pod at solidweb.me 😉
+        </ExternalButtonLink>
+        <br />
+        It runs the more modern Community Solid Server
+      </>
+    ),
+  },
+  {
+    id: 'more-control',
+    label: 'I want more control',
+    content: (
+      <>
+        <p>
+          You can <b>host your Pod</b>. This will give you full control over
+          your data. However, it's more complicated to set up, and requires
+          maintenance. We recommend{' '}
+          <a
+            href="https://github.com/communitysolidserver/communitysolidserver#-running-the-server"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Community Solid Server
+          </a>
+          .
+        </p>
+        <p>
+          <i>
+            Read more about{' '}
+            <a
+              href="https://solidproject.org/self-hosting/css"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              running your own Solid server
+            </a>
+          </i>
+        </p>
+        <hr />
+        <p>
+          If you own a <b>domain name</b>, you can have your webId on your
+          domain. This gives you more control over your Solid identity.
+        </p>
+        <p>
+          Choose option <i>"Use my existing WebID to access my Pod"</i> with one
+          of these providers
+        </p>
+        <ul>
+          {oidcIssuers
+            .filter(({ server }) => server === 'CSS')
+            .map(({ issuer, registration }) => (
+              <li>
+                <ExternalButtonLink
+                  secondary
+                  href={registration}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {new URL(issuer).hostname}
+                </ExternalButtonLink>
+              </li>
+            ))}
+        </ul>
+        (this will need more explanation),
+        <br />
+        or host your own Solid Server.
```

Comment:
Not sure if running own Solid Server is one of the 3 options under _'I want more control'_ or a sub-option under owning your identity (I guess could be both), in the former case (which I slightly prefer as there are less nested levels in the decision tree), there is \<hr /> missing ;)

Reply:
It's meant as a sub-option under owning your identity. I'll try to clarify it somehow.

Reply:
Do you think it conveys the message more clearly now? (i also tried to briefly include some other specifics)